modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/DXPCloud/index.js

```diff
@@ -148,7 +148,7 @@ const ActivationStatusDXPCloud = ({
 			title: 'Activation Status',
 		},
 		[STATUS_TAG_TYPE_NAMES.inProgress]: {
-			dropdownIcon: userAccount.isStaff && (
+			dropdownIcon: userAccount.isStaff && userAccount.isProvisioning && (
```

Comment:
I guess the stories need to be staff or provisioning user role.

```suggestion
			dropdownIcon: (userAccount.isStaff || userAccount.isProvisioning) && (
```